src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/ReferencePopulationDefinitionFreeText.java

```diff
@@ -0,0 +1,82 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import org.apache.commons.lang3.StringUtils;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+import java.util.Arrays;
+
+/**
+ * Example: {@code FTX+RPD+++Equivocal'}
+ */
+@EqualsAndHashCode(callSuper = false)
+@Getter
+@AllArgsConstructor
+public class ReferencePopulationDefinitionFreeText extends Segment {
+    private static final String KEY = "FTX";
+    private static final String QUALIFIER = "RPD";
+    private static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
+
+    private static final int INDEX_FREE_TEXTS = 4;
+    private static final int MAXIMUM_FREE_TEXTS = 5;
+
+    @NonNull
+    private final String[] freeTexts;
+
+    public static ReferencePopulationDefinitionFreeText fromString(final String edifact) {
+        if (!edifact.startsWith(KEY_QUALIFIER)) {
+            throw new IllegalArgumentException("Can't create "
+                + ReferencePopulationDefinitionFreeText.class.getSimpleName() + " from " + edifact);
+        }
+
+        final String[] split = Split.byPlus(edifact);
+        final String freeTextComponent = split[INDEX_FREE_TEXTS];
+
+        final String[] freeTexts = Split.byColon(freeTextComponent);
+        if (freeTexts.length > MAXIMUM_FREE_TEXTS) {
+            throw new IllegalArgumentException("Can't create "
```

Comment:
Could we make the message more descriptive? Add something like: `...free texts length exceeds max limit`

Reply:
We could of course! I don't see any precedents for it though, in our or NHAIS' code. Happy to break precedent if the team agrees.

The only other note I can make on this is that we can still find out which of the two exceptions was thrown by looking at the line numbers in the stacktrace, but that is certainly more awkward than just seeing the problem in the message!

Reply:
I'd be in favour of having a more descriptive error message like the one in `preValidate()` for the second condition as this is the same condition as the one in `preValidate()` so if we want to keep the messages consistent, I think using that message would be good. 

```suggestion
            throw new IllegalArgumentException("KEY_QUALIFIER + ": At most " + MAXIMUM_FREE_TEXTS + " reference population definitions may be given."
 "
```

Reply:
Yeah I think more descriptive error messages would be good (here and across our other parsers). I don't think we should tie ourselves too much to how NHAIS works

Reply:
I agree as well that more descriptive error messages are better. It definitely doesn't hurt us :)